### ISSUE TITLE
Warnings on CardImage

### ISSUE THREAD

**Issue by asciifaceman** [sidevesh/react-native-material-cards#11]
<img width="364" alt="screen shot 2017-12-28 at 11 59 38 pm" src="https://user-images.githubusercontent.com/1619591/34432354-4c35af32-ec2b-11e7-8015-86d51106f6f7.png">
<img width="357" alt="screen shot 2017-12-29 at 12 00 02 am" src="https://user-images.githubusercontent.com/1619591/34432355-4c4cd0d6-ec2b-11e7-9170-80d20e078ff3.png">

Offending line:
```
<Image source={this.props.source} resizeMode={["stretch", this.props.resizeMode]} resizeMethod={["resize", this.props.resizeMethod]} style={[styles.imageContainer,  {height: this.state.calc_height}]}>
```

Line 20, CardImage.js

**Comment by wunderkind2k1** [comment]
Maybe this is related to the react-native version? Is your version newer than 0.47.1? Since than you should use ImageBackground instead of image. See this commit https://github.com/facebook/react-native/commit/9637dd4a1b78d5f6da510b1b4ad10d45d67badbf#diff-20b0985c988906f7757e8b2f340ecdbb - I have an open pull request https://github.com/SiDevesh/React-Native-Material-Cards/pull/8 that solved my problem.

**Comment by asciifaceman** [comment]
Yep I just wasn't in a position to open a PR. Also I'm still iffy on ImageBackground because of how gimped it is, but yeah many tools are now on 0.6

**Comment by sidevesh** [comment]
Have merged #8, should be fixed now, Sorry for delay.
